app/src/main/java/net/skyscanner/backpack/demo/compose/BottomSheetStory.kt

```diff
@@ -105,3 +227,20 @@ fun SheetContent(modifier: Modifier = Modifier) {
         }
     }
 }
+
+@Composable
+fun ImageContent(modifier: Modifier = Modifier) {
+    BpkFlare(
```

Comment:
Sorry I should've been clearer. Rather than this & backpack logo can you use one of the images, like `beach`  and render that in just an `Image`as that matches what figma shows more closely.

Reply:
Updated!